src/components/Loading/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+function Loading() {
+  // console.log('testing12312312')
```

Comment:
remove console
run prettier/lint in case

Reply:
LOL yes, this is the other one you mentioned that i forget